Client/Frontend/Browser/Tab Tray/Views/TabSyncContainerView.swift

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+import BraveUI
+
+extension TabTrayController {
+
+  class TabSyncContainerView: UIView {
+    
+    // MARK: UX
+
+    struct UX {
+      static let sectionTopPadding = 5.0
+    }
+
+    var tableView = UITableView()
+    
+    lazy var noSyncTabsOverlayView = EmptyStateOverlayView(
```

Comment:
nit: can be `private(set) `

Reply:
Fixed at https://github.com/brave/brave-ios/pull/5809/commits/8f2607664aadd1db8ecadc85dd584cb748433a79